firmware/fw_base.S

```diff
@@ -119,7 +119,7 @@ _prev_arg1_override_done:
 	 */
 	beqz	a1, _fdt_reloc_done
 	/* Mask values in a3 and a4 */
-	li	a3, ~0xf
+	li	a3, ~(__SIZEOF_POINTER - 1)
```

Comment:
Typo here. Use __SIZEOF_POINTER__ instead of __SIZEOF_POINTER

Reply:
Done